run-tests.sh

```diff
@@ -6,9 +6,39 @@
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+export REANA_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://postgres:mysecretpassword@localhost/postgres
+
+# Verify that db container is running before continuing
+check_ready() {
+    RETRIES=40
+    while ! $2
+    do
+        echo "Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+_db_check(){ docker exec --user postgres reana-postgres bash -c "pg_isready" &>/dev/null; }
+
+
 pydocstyle reana_db && \
 black --check . && \
 check-manifest --ignore ".travis-*" && \
 sphinx-build -qnNW docs docs/_build/html && \
-REANA_SQLALCHEMY_DATABASE_URI=sqlite:// python setup.py test && \
+OLD="$(docker ps --all --quiet --filter=name=reana-postgres)"
+if [ -n "$OLD" ]; then
+  echo 'Cleaning old DB container...' && \
+  docker stop reana-postgres
+fi
+echo 'Starting DB container...' && \
+docker run --rm --name reana-postgres -p 5432:5432 -e POSTGRES_PASSWORD=mysecretpassword -d postgres && \
+check_ready "Postgres" _db_check && \
+python setup.py test && \
+echo 'Stopping DB container...' && \
```

Comment:
Since the script has now grown and we now have separate functions, can you please amend it as follows:

- create functions for everything
- introduce newlines between them
- use echo statements everywhere (e.g. 'Checking Black compliance...') for consistency
- the container name is indeed called "postgres", but the DB name is "PostgreSQL", better use that one
- we can use '==> [INFO]' and '==> [ERROR]' for messages that we create ourselves